### ISSUE TITLE
Prompts being used as tokens to activate trained subject instead of target token

### ISSUE THREAD

**Issue by Pychnight** [d8ahazard/sd_dreambooth_extension#1166]
## ⚠️If you do not follow the template, your issue may be closed without a response ⚠️
Kindly read and fill this form in its entirety.

#######################################################################################################
Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: a9fed7c364061ae6efb37f797b6b522cb3cf7aa2

Dreambooth revision: bd3fecc3d27d777a4e8f3206a0b16e852877dbad
SD-WebUI revision: 
Initializing Dreambooth
Dreambooth revision: 926ae204ef5de17efca2059c334b6098492a0641
Successfully installed accelerate-0.18.0 fastapi-0.94.1 gitpython-3.1.31 transformers-4.26.1

[+] xformers version 0.0.17.dev464 installed.
[+] torch version 1.13.1+cu117 installed.
[+] torchvision version 0.14.1+cu117 installed.
[+] accelerate version 0.18.0 installed.
[+] diffusers version 0.14.0 installed.
[+] transformers version 4.26.1 installed.
[+] bitsandbytes version 0.35.4 installed.

#######################################################################################################
```


### 2. Describe the bug

After doing some training tests on simple subjects (stop signs and trees)

I noticed that my token is not activating the token trained but when i use the prompt it activates the image trained, even when it generates samples from the token they are very off.

Also, i was wondering would it be possible to implement better multi-concept isolation, so we can train multi-concept subjects without them blending together.

**Screenshots/Config**
If the issue is specific to an error while training, please provide a screenshot of training parameters or the
db_config.json file from /models/dreambooth/MODELNAME/db_config.json

### 3. Provide logs

If a crash has occurred, please provide the *entire* stack trace from the log, including the last few log messages *before* the crash occurred.

```st
PASTE YOUR STACKTRACE HERE
```

### 4. Environment

What OS?
Windows 11

If Windows - WSL or native?
Native

What GPU are you using?
gigabyte 4090 OC

**Comment by saunderez** [comment]
You've overtrained the text encoder. The classic sign of this is a model that will only sometimes output the subject with the instance token, but will always output the subject with captions from the dataset. Another sign is generated images frequently having the same layout, but different visuals. For instance a portrait that always has a road behind it and a park, when this occurs it can be traced back to dataset captions.

You can prevent this in future trainings by decreasing the text encoder ratio, decreasing the text encoder learning rate if using LORA, stopping training earlier or increasing weight decay (use this as last resort as it will aso imapct the unet quality)
